src/main/java/org/example/algorithms/CheckAnagram.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Arrays;
+import java.util.Scanner;
+public class CheckAnagramOrNot

```

Comment:
Name of public class should be same as file name or the program does not compile